index.js

```diff
@@ -1 +1,47 @@
 // implement your API here
+const express = require('express');
+const db = require('./data/db');
+
+const server = express();
+
+server.use(express.json());
+
+server.post('/api/users', (req, res) => {
+	if (!req.body.name || !req.body.bio) {
```

Comment:
I would reverse the order of if and else. ONLY give 201 if you have both name or bio. add another conditions if you are missing bio or name and a catch if anything else goes wrong